ipa-server-configure-first

```diff
@@ -208,6 +208,14 @@ else
 			fi
 		done
 		cp /etc/volume-version /data/volume-version
+
+		if ! [ -f /etc/ipa/ca.crt ] && [ -f /root/ipa.csr ] ; then
+			cp -p /root/ipa.csr /data/ipa.csr
+			echo "FreeIPA server certificate signing request file is in volume in file ipa.csr."
```

Comment:
Sounds good .. but would we be able to shorten the message to < 80 characters in length?